models/person/person.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/local/bin/python3
+class Person(object):
+    """The person class."""
+
+    id = 0
```

Comment:
`id` is a built in function in Python.. assigning a variable to it overwrites it..